src/components/atoms/CircleProgress/CircleProgress.tsx

```diff
@@ -144,7 +144,13 @@ export default function CircleProgress({
     const progressClass = getProgressClass(completion);
     const { arcRotation, radius } = initData;
     const circleStyles = getCircleStyles(radius, arcRotation, completion);
-    return { circleStyles, completion, progressClass: progressClass };
+    const circleBackgroundStyles = getCircleStyles(radius, arcRotation, 100);
```

Comment:
Calling the function again does the exact same calculation (when completion is 100, `strokeDashoffset`=`rotationOffset`). So rather than calling the function again, return `rotationOffset` from it and generate the style objects here.